ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -775,7 +775,8 @@ static int component_select(struct ompi_win_t *win, void **base, size_t size, in
     case MPI_WIN_FLAVOR_CREATE:
         mem_type = OPAL_COMMON_UCX_MEM_MAP;
         break;
-    case MPI_WIN_FLAVOR_SHARED:
+    default:
```

Comment:
IMO better would be to keep the case for `MPI_WIN_FLAVOR_SHARED` and add a default that signals an error if the flavor is not known.

Reply:
ok, fixed